### PR TITLE
Update monorepo to 4.5.0.

### PR DESCRIPTION
**Pull request by mattsoulanille**

### CODE REVIEW

File: tfjs-backend-wasm/starter/parcel/package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
   },
   "dependencies": {
     "@tensorflow/tfjs-backend-wasm": "3.14.0",
-    "@tensorflow/tfjs-core": "3.14.0"
+    "@tensorflow/tfjs-core": "4.5.0"
```
Comment (chunnienc) [inline tfjs-backend-wasm/starter/parcel/package.json:11]: why do we still update this version? I thought it's fixed in the previous version updater PR.

**Review by chunnienc — CHANGES_REQUESTED** [review]

File: tfjs-backend-wasm/starter/parcel/package.json (hunk 1 above)
Comment (mattsoulanille) [reply tfjs-backend-wasm/starter/parcel/package.json:11]: Oh, #7550 hasn't been merged yet. Let me try again on that branch.
